### PR TITLE
New sample app and new ReadCSV method in Util.cs

### PR DESCRIPTION
**Pull request by ShacharWeis**
It's good stuff.

### CODE REVIEW

File: SampleConsoleApplication/SampleConsoleApplication.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SampleConsoleApplication
+{
+    class SampleConsoleApplication
+    {
+        static string API_TOKEN = "cw5EdmA2M8MiKKQJnFud";
```
Comment (danmcc) [inline SampleConsoleApplication/SampleConsoleApplication.cs]: Can we replace these with environment vars, or something passed in on the command line?  These are used just for internal testing.

File: SampleConsoleApplication/SampleConsoleApplication.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    throw new Exception("Authentication failed. Please check that you have the correct API token and KEY");
+
+                Console.WriteLine("Authentication veridied");
+
+                // Lets load a CSV file
+                IEnumerable<ListItem> listItems = Util.LoadCSV("sample.csv");
+
+                ListItem test = new ListItem();
+                test.SuggestedScore = 1;
+                test["suggested_score"] = 1;
```
Comment (danmcc) [inline SampleConsoleApplication/SampleConsoleApplication.cs]: Is this to illustrate different ways of adding data, or just experimentation?

**Comment by ShacharWeis** [comment]
Oops, I was going to remove those and of course forgot. I'll update it
tomorrow.

**Comment by danmcc** [comment]
Looks good!  Thanks.

**Pull request merged**